docs/content/how-to/kubevirt/external-infrastructure.md

```diff
@@ -59,6 +59,9 @@ The user or service account used in the provided kubeconfig should have full per
 * `virtualmachineinstances.kubevirt.io`
 * `datavolumes.cdi.kubevirt.io`
 * `services`
+* `endpointslices`
+* `endpointslices/restricted`
+* `machines/finalizers`
```

Comment:
the owner reference handles the endpointslice removal when the machine disappears, right? seems like we shouldn't need finalizers.

Reply:
When we set the owner ref using `ownerRef := config.OwnerRefFrom(machine)` it also sets the `blockOwnerDeletion` and for that we we need the RBAC

https://sdk.operatorframework.io/docs/faqs/#after-deploying-my-operator-why-do-i-see-errors-like-is-forbidden-cannot-set-blockownerdeletion-if-an-ownerreference-refers-to-a-resource-you-cant-set-finalizers-on-

Maybe we don't need `blockOwnerDeletion` ?